docs/BR.md

```diff
@@ -1190,15 +1190,36 @@ The certificate request MAY include all factual information about the Applicant
+[Section 6.3.2](#632-certificate-operational-periods-and-key-pair-usage-periods) limits the validity period of Subscriber Certificates.
+
+The CA MAY use the documents and data provided in [Section 3.2](#32-initial-identity-validation) to verify certificate information, or may reuse previous validations themselves, provided that the CA obtained the data or document from a source specified under [Section 3.2](#32-initial-identity-validation) or completed the validation itself within the maximum number of days prior to issuing the Certificate, as defined in the following table:
+
+Table: Maximum Allowed Non-SAN Validation Data Reuse Period
+
+| __Certificate issued on or after__ | __Certificate issued before__ | __Maximum number of days for data reuse__ |
+| --                                 | -                             | -                                         |
+|                                    | September 15, 2025            | 825                                       |
+| September 15, 2025                 | September 15, 2026            | 366                                       |
+
+For validation of Domain Names and IP Addresses according to [Section 3.2.2.4](#3224-validation-of-domain-authorization-or-control) and [Section 3.2.2.5](#3225-authentication-for-an-ip-address), any data, document, or completed validation used MUST be obtained within the maximum number of days days prior to issuing the Certificate, as defined in the following table:
+
+Table: Maximum Allowed SAN Validation Data Reuse Period
+
+| __Certificate issued on or after__ | __Certificate issued before__ | __Maximum number of days for data reuse__ |
+| --                                 | -                             | -                                         |
+|                                    | September 15, 2025            | 398                                       |
+| September 15, 2025                 | September 15, 2026            | 200                                       |
+| September 15, 2026                 | April 15, 2027                | 100                                       |
+| April 15, 2027                     | September 15, 2027            | 45                                        |
+| September 15, 2027                 |                               | 10                                        |
```

Comment:
suggestion: stick to just 3 effective dates, on Sept 15 of the next three years. I think the inclusion of the April date makes it harder to think about this ballot's timeline, without providing a commensurate benefit.
```suggestion
|                                    | September 15, 2025            | 398                                       |
| September 15, 2025                 | September 15, 2026            | 200                                       |
| September 15, 2026                 | September 15, 2027            | 100                                       |
| September 15, 2027                 |                               | 10                                        |
```